src/main/java/de/tum/in/www1/artemis/web/rest/UserResource.java

```diff
@@ -274,10 +284,37 @@ public ResponseEntity<UserDTO> getUser(@PathVariable String login) {
     @PreAuthorize("hasRole('ADMIN')")
     public ResponseEntity<Void> deleteUser(@PathVariable String login) {
         log.debug("REST request to delete User: {}", login);
+        if (isCurrentUser(login)) {
+            throw new BadRequestAlertException("You cannot delete yourself", "userManagement", "cannotDeleteYourself");
+        }
         userService.deleteUser(login);
         return ResponseEntity.ok().headers(HeaderUtil.createAlert(applicationName, "userManagement.deleted", login)).build();
     }
 
+    /**
+     * Delete users: deletes the provided users
+     *
+     * @param logins user logins to delete
+     * @return the ResponseEntity with status 200 (OK)
+     */
+    @DeleteMapping("users")
+    @PreAuthorize("hasRole('ADMIN')")
+    public ResponseEntity<List<String>> deleteUsers(@RequestParam(name = "login") List<String> logins) {
+        log.debug("REST request to delete {} users", logins.size());
+        List<String> deletedUsers = new java.util.ArrayList<>();
+        for (String login : logins) {
+            try {
+                if (!isCurrentUser(login)) {
+                    userService.deleteUser(login);
+                    deletedUsers.add(login);
+                }
+            }
+            catch (Exception ignored) {
+            }
```

Comment:
maybe you should also add a comment why this can be ignored (e.g., that logging is done or something else. not sure.